src/main/java/github/scarsz/discordsrv/objects/managers/GroupSynchronizationManager.java

```diff
@@ -178,6 +178,11 @@ public void resync(OfflinePlayer player, SyncDirection direction, boolean addLin
 
         Map<Guild, Map<String, Set<Role>>> roleChanges = new HashMap<>();
 
+        // Check if Minecraft or Discord is strictly authoritative.
+        boolean oneWaySynchronisation = DiscordSRV.config().getBoolean("GroupRoleSynchronizationOneWay");
+        boolean minecraftIsStrictlyAuthoritative = oneWaySynchronisation && DiscordSRV.config().getBoolean("GroupRoleSynchronizationMinecraftIsAuthoritative");
+        boolean discordIsStrictlyAuthoritative = oneWaySynchronisation && !DiscordSRV.config().getBoolean("GroupRoleSynchronizationMinecraftIsAuthoritative");
```

Comment:
Should log these to `synchronizationSummary`, eg. `Synchronization is one way to Minecraft`

Reply:
Done!